### ISSUE TITLE
The live update feature does not show the latest status correctly.

### ISSUE THREAD

**Issue by raindropsfromsky** [osmandapp/Osmand#3622]
With the live map update feature, we may want to know the following things:
1. Which is the latest version on the server? (date+time)
2. Which is the latest local version? (date+time)
3. What is the pending update size? (we can estimate how much time will it take to download)
4. What is the cumulative update size? (this much data is stored separately from the map obf file).

However, the current design shows only 1 and 3 (or is it 1 and 4?)
![liveupdate](https://cloud.githubusercontent.com/assets/9047168/24643143/9342b814-1929-11e7-82b3-4378f17615c8.png)

This creates a confusion. For example, 
1. The screenshot above shows a last updated timestamp that is more than 24 hours old. </br>In practice, the latest changes I made in OSM yesterday afternoon have not appeared in OsmAnd. </br>Why was the map not updated on server?
2. From the display, I cannot make up my mind whether my local copy is fully updated. 
3. If I tap on **Update now**, the display does not change at all. </br> There is no pop up feedback message. What should I conclude?
4. The display shows the Update size as 34 MB. </br> Is this the size of pending update or the cumulative size of the past updates in my local memory?

**Desired:** Correct the fields to clear these doubts.

**Comment by vshcherb** [comment]
1. Which is the latest version on the server? (date+time)
If you click Update now, you will get the latest version on the server or error if you don't have internet connection
2. Which is the latest local version? (date+time)
- Latest map change is the latest date what's been changed in the map. It doesn't show what's the latest date processed map.
3. What is the pending update size? (we can estimate how much time will it take to download)
No information, though usually they are small. They are also coming in batch of files.
4. What is the cumulative update size? (this much data is stored separately from the map obf file).
Update size is the cumulative size of all updates not including base map itself.

**Comment by raindropsfromsky** [comment]
Hi Viktor,

Sorry for the late response. But I was actually not looking for the answers for myself. The point is that the GUI is ambiguous, and leaves so many questions unanswered.

Ok rather than leaving this open-ended, I will go ahead and suggest some GUI changes. 

1. Just below the **Last map change:** field, show **Last updated:** field. </br>by comparing the two, we know if there is an update pending.
2. Instead of **Update size:**, reword it to **Total local updates:**</br>That means we have accumulated this much local data to update the original obf file.
3. If the latest update is already downloaded, then gray out the **Update now** button. </br> It should be active only when some update is pending.
4. There are two buttons, **Cancel** and **OK**. But both are functionally same. </br> In fact, there is no ongoing operation to cancel. </br>So both buttons do the same thing: Close the dialog box and return to previous menu. </br>Therefore both buttons should be replaced with a **Close** button.
